api/src/test/java/org/openmrs/api/impl/PatientServiceImplTest.java

```diff
@@ -3,55 +3,69 @@
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
+ * 
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-package org.openmrs.api.impl;
-
-import static junit.framework.TestCase.assertTrue;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
-import static org.mockito.ArgumentMatchers.any;
```

Comment:
Please look at https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Imports
Not sure which IDE you are using but for example for IntelliJ https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+IntelliJ#DeveloperHow-ToSetupAndUseIntelliJ-JavaImports

You can see that static imports should go to the top followed by a newline. After that we group the java, javax imports, org and then the rest.